twake/backend/node/config/default.json

```diff
@@ -74,5 +76,18 @@
   },
   "pubsub": {
     "urls": ["amqp://guest:guest@localhost:5672"]
+  },
+  "storage": {
+    "secret": "tp63bb3e90c0271c9a1c06651a7c0967eab8851a7a897769",
+    "type": "S3", 
+    "S3": {
+
+    },
+    "local": {
+
+    },
+    "swift": {
+
+    }
```

Comment:
There should be default values here.
- For S3 you can put what you currently have without the api id and secret (replace it by admin/admin)
- For local you can put one folder:
   {
      "path": "/drive/"
   }